examples/rust/digital_signature/proof/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Risc0, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg_attr(not(test), no_std)]
+
+use r0vm_core::Digest;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct Message {
+    pub msg: u32,
```

Comment:
Maybe we should make messages actual a digest themselves or at least a something big enough to hold a digest (like 32 bytes of whatever)?  I realize it's only an example, but it's an easy change

Reply:
Fixed in latest commit; message and pass are now both 32 *bytes*.